tools/crm_resource_runtime.c

```diff
@@ -1710,6 +1710,14 @@ cli_resource_execute_from_params(pcmk__output_t *out, const char *rsc_name,
         free(level);
     }
 
+    if (pcmk__str_eq(out->fmt_name, "xml", pcmk__str_none)) {
+        resource_verbose = 1;
```

Comment:
I assume this was just for testing?

Reply:
No, that was there to skip the short circuit on hiding validate-all output.  However, we probably don't want to set HA_debug and OCF_TRACE_RA, so I can come up with another way to do this.

Reply:
Fixed.